### PR TITLE
Fixed import

### PR DESCRIPTION
**Pull request by arijit-ship**
Updated import. `get_sparse_operator` is now moved to `openfermion.linalg` and `MolecularData` is now moved to `openfermion.chem`

### CODE REVIEW

**Review by KowerKoint — APPROVED** [review]
Thank you.
We merge this after #610 is merged.

**Comment by ikanago** [comment]
Looks the same as #615. Thanks!

**Pull request closed without merging**
